build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
Seems like this can be controlled with the `USER` directive.
```
When using docker exec for a container which is using USER something in the Dockerfile, the command is also executed as this user.
```
Would that support your use case?